BrightID/src/App.tsx

```diff
@@ -27,8 +27,8 @@ export const App = () => {
       screens: {
         App: {
           screens: {
-            Apps: {
-              path: 'link-verification/:baseUrl?/:context/:contextId/',
+            NodeModal: {
```

Comment:
I think this is an error left from debugging. Why do you remove the Apps route?

Reply:
You're right! thanks for your eyes on this. fixed :)